### PR TITLE
Node - Week5 Assignment

### PR DESCRIPTION
**Pull request by AnnaSolovykh**

### CODE REVIEW

**Review by leo-mutuku — APPROVED** [review]
database connection well imported

env variables well called

File: 03-task-manager/starter/controllers/tasks.js
Comment (leo-mutuku) [inline 03-task-manager/starter/controllers/tasks.js:1]: Task model well imported, controller CRUDE operation well implimented

File: 03-task-manager/starter/models/Task.js
Comment (leo-mutuku) [inline 03-task-manager/starter/models/Task.js:1]: Mongoose schema class well instagiated, task schema created correctly

File: 03-task-manager/optionalArrayMethodsReviewExtraAssignment.js
Comment (leo-mutuku) [inline 03-task-manager/optionalArrayMethodsReviewExtraAssignment.js:1]: Array method well implimented

**Review by leo-mutuku — APPROVED** [review]
Hi Anna, the task implimentation is alright. You seem to had some challenges with your git repo, week4 PR seem to have disapper! However thing is okay you can proceed. Incase you need an extra hand for the git let me know.

**Comment by AnnaSolovykh** [comment]
@leo-mutuku 
Screen Shot 2023-11-03 at 13 30 54
Hi! Thank you for the review. As for branch 4, I merged it into main and deleted it.

**Comment by leo-mutuku** [comment]
No problem that's well noted.

**Comment by AnnaSolovykh** [comment]
Anyway I have restored it :) Let it be!

**Comment by leo-mutuku** [comment]
Great!

**Pull request merged**